service/src/main/java/edu/tamu/catalog/controller/CatalogAccessController.java

```diff
@@ -64,6 +68,24 @@ public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="e
         }
     }
 
+    /**
+     * Provides data for all fees and fines associated with a patron.
+     *
+     * @param String catalogName (optional)
+     * @param String user
+     * @return
+     */
+    @GetMapping("/fines")
+    public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="folio") String catalogName, @WeaverUser User user) {
+        FeesFines FeesFines = getCatalogServiceByName(catalogName).getFeesFines(user.getUsername());
+
+        if (FeesFines != null) {
```

Comment:
Probably want to avoid variable name exactly same as type. Was confused by this for a moment.

Reply:
This null check will never occur for the current implementation of `getFeesFines`.

Reply:
`FeesFines FeesFines` was meant to be `FeesFines feesFines`.

Reply:
That is what I was looking for and I was like "how is this class not null".